tmt-app/src/Components/Quiz.jsx

```diff
@@ -29,6 +28,27 @@ function Quiz() {
           </Grid>
         ))}
       </Grid>
+      <Link
```

Comment:
We need some layout for this button. What does the desktop and mobile positions look like for this?

Reviewing on mobile is pretty unusable still

Reply:
This was going to be a temporary button to navigate through the quiz without having to change url. I will make the adjustment to make it the permanent now.

Reply:
Sounds good, yeah if we're gonna add a temp button might as well put in the few extra steps for the one matching the design spec